docs/ops.html

```diff
@@ -4045,7 +4045,7 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
             <li>
               On each broker, remove the <code>zookeeper.metadata.migration.enable</code>,
               <code>controller.listener.names</code>, and <code>controller.quorum.voters</code>
-              configurations, and replace <code>node.id</code> with <code>broker.id</code>.
+              configurations.
```

Comment:
Good catch! Should we do the same thing for `Migrating brokers to KRaft` state? 

I'm thinking in the 1st step of `Migrating brokers to KRaft`, it should add `replace the broker.id with node.id`, ex:
```
On each broker, remove the <code>process.roles</code>,  replace the <code>node.id</code> with <code>broker.id</code>, and restore the...
```

And then, in the last step , we can also remove the `and replace <code>node.id</code> with <code>broker.id</code>` sentence as above. WDYT?

Reply:
I think you are right! The first step is to come back to the "Enter Migration Mode on the brokers" in the end, then following the same steps.